### DISCUSSION TITLE
ydb start error

### DISCUSSION THREAD

**Discussion by wellyn-chen** [seaweedfs/seaweedfs#3134] [Q&A]
![image](https://user-images.githubusercontent.com/88520903/171384475-90cc37cb-df99-4453-9072-07ccbba7d7b4.png)

**Comment by wellyn-chen** [comment]
Supported filer stores are:
    arangodb
    cassandra
    etcd
    hbase
    leveldb
    leveldb2
    leveldb3
    mongodb
    mysql
    mysql2
    postgres
    postgres2
    redis_cluster
    redis
    redis_cluster2
    redis2_sentinel
    redis2
    redis_cluster3
    redis3_sentinel
    redis3

**Comment by kmlebedev** [comment]
@wellyn-chen  Please use full binary https://github.com/chrislusf/seaweedfs/releases/download/3.08/linux_amd64_full.tar.gz
or image with tag [3.08_full](https://hub.docker.com/layers/seaweedfs/chrislusf/seaweedfs/3.08_full/images/sha256-4feff6a277098af87da649b9ac3ce744d19d09fe869e1db8c8dffea217cb9c89?context=explore)
